enterprise/backend/test/metabase_enterprise/serialization/v2/extract_test.clj

```diff
@@ -225,24 +227,26 @@
                        :model/Card       {c4-id  :id
                                           c4-eid :entity_id}        {:name          "Referenced Question"
                                                                      :database_id   db-id
-                                                                     :table_id      schema-id
+                                                                     :table_id      no-schema-id
                                                                      :collection_id coll-id
                                                                      :creator_id    mark-id
                                                                      :dataset_query
                                                                      {:query {:source-table no-schema-id
                                                                               :filter [:>= [:field field-id nil] 18]}
-                                                                      :database db-id}}
+                                                                      :database db-id
+                                                                      :type :query}}
                        :model/Card
                        {c5-id  :id
                         c5-eid :entity_id}
                        {:name          "Dependent Question"
                         :database_id   db-id
-                        :table_id      schema-id
+                        :table_id      no-schema-id
```

Comment:
So this card is based on Card `c4` which uses the `no-schema-id` table... so the old `table_id` value was wrong. Not sure why the code to populate this stuff wasn't doing it before but we were always supposed to be ignoring the `database_id` and `table_id` you set for a Card and populate that stuff based on the `dataset_query`